README.md

```diff
@@ -30,14 +38,56 @@ export DATABASE_NAME=jehlomat
 export JWT_ISSUER=http://localhost:8082/
 export JWT_AUDIENCE=http://localhost:8082/
 export JWT_REALM=jehlomat_local_realm
-export SUPER_ADMIN_EMAIL=jehlomat@cesko.digital
```

Comment:
These env variables are still necessary, why do you want to delete from readme?

Reply:
https://github.com/cesko-digital/jehlomat/pull/163/commits/1a5d6f732238dd2f8c14ee0dca34cc252b3157c2